ui/src/me/components/LogoutButton.tsx

```diff
@@ -4,30 +4,12 @@ import {Link} from 'react-router'
 
 // Components
 import {Button, ComponentSize} from '@influxdata/clockface'
-import CloudExclude from 'src/shared/components/cloud/CloudExclude'
-import CloudOnly from 'src/shared/components/cloud/CloudOnly'
-import {CLOUD_URL, CLOUD_LOGOUT_PATH} from 'src/shared/constants'
-import {FeatureFlag} from 'src/shared/utils/featureFlag'
 
 const LogoutButton: SFC = () => (
   <>
-    <FeatureFlag name="regionBasedLoginPage">
```

Comment:
In the process of this revision, I revisited the LogoutButton functionality and ensured that the existing `Logout` component functions identically, removing the need to conditionally render routes based on `CloudExclude` and `CloudOnly`